src/cmdlib.sh

```diff
@@ -790,28 +790,16 @@ EOF
     esac
 
     kola_args=(kola qemuexec -m "${COSA_SUPERMIN_MEMORY:-${memory_default}}" --auto-cpus -U --workdir none \
-               --console-to-file "${runvm_console}")
+               --console-to-file "${runvm_console}" --bind-rw "${workdir},workdir")
 
     base_qemu_args=(-drive 'if=none,id=root,format=raw,snapshot=on,file='"${vmbuilddir}"'/root,index=1' \
                     -device 'virtio-blk,drive=root' \
                     -kernel "${vmbuilddir}/kernel" -initrd "${vmbuilddir}/initrd" \
                     -no-reboot -nodefaults \
                     -device virtio-serial \
-                    -virtfs 'local,id=workdir,path='"${workdir}"',security_model=none,mount_tag=workdir' \
                     -append "root=UUID=${superminrootfsuuid} console=${DEFAULT_TERMINAL} selinux=1 enforcing=0 autorelabel=1" \
                    )
 
-    # support local dev cases where src/config is a symlink.  Note if you change or extend to this set,
-    # you also need to update supermin-init-prelude.sh to mount it inside the VM.
-    for maybe_symlink in "${workdir}"/{src/config,src/yumrepos,builds}; do
```

Comment:
AFAICT in testing, we still need the virtiofs equivalent of this, no?

Reply:
Yes, you're right.  Fixed.